version/1/2/0/code_of_conduct.md

```diff
@@ -0,0 +1,23 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or political affiliation.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexual language or imagery
+* Derogatory comments
+* Personal attacks
+* Trolling or insulting comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic addresses, without permission
+* Other unethical or unprofessional conduct.
```

Comment:
> I would like to see these loopholes closed.

That would be nice, but there will *always* be room for some interpretation, now matter how hard you try to make fuzzy lines bright and hard.

> I disagree that enumerating behavior or clarifying language is futile

Okay, I don't think adding more examples or clarifying language is futile, only that trying to make the list fully comprehensive is. After a while enumerating more hits diminishing returns, after which you will still need "Including but not limited to" language.

We may have reached the point where you and I will have to agree to disagree.